src/facade/command_id.h

```diff
@@ -102,6 +112,9 @@ class CommandId {
 
   // Whether the command can only be used by admin connections.
   bool restricted_ = false;
+
+  bool is_pub_sub_ = false;
+  bool is_p_sub_ = false;
 };
```

Comment:
nit: not sure we need to store this

Reply:
we don't we can string match but I would like to avoid it the bool's have almost zero impact and they are faster than string matching :)